src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,115 @@
+            ignored.
+            * "hardest": cadences with all the above flags will be ignored, in addition to cadences with GSFC-ELEANOR-LITE
+            bit flags of 17 (decimal value 131072) and 18 (decimal value 262144).
+    """
+
+    lc = read_generic_lightcurve(
+        filename,
+        time_column="TIME".lower(),
+        flux_column=flux_column.lower(),
+        flux_err_column = "FLUX_ERR".lower(),
+        time_format="btjd",
+        quality_column= "QUALITY".lower(),
+        centroid_col_column = "X_CENTROID".lower(),
+        centroid_row_column = "Y_CENTROID".lower(),
+        cadenceno_column = "FFIINDEX".lower()
+    )
+
+    if quality_bitmask == "hardest":
+        # Eleanor has 2 additional bits on top of the 16 TESS SPOC bits
+        # they are excluded when hardest is specified.
+        quality_bitmask = TessQualityFlags.HARDEST_BITMASK | 2** 17 | 2**18
+
+    quality_mask = TessQualityFlags.create_quality_mask(
+        quality_array=lc["quality"], bitmask=quality_bitmask
+    )
+
+    lc = lc[quality_mask]
+
+    # Eleanor FITS file do not have units specified. re-add them.
+    for colname in ["flux", "flux_err", "raw_flux", "corr_flux", "pca_flux", "psf_flux"]:
+        if colname in lc.colnames:
+            if lc[colname].unit is not None:
+                # for case flux, flux_err, lightkurve has forced it to be u.dimensionless_unscaled
+                # can't reset a unit, so we create a new column
+                lc[colname] = u.Quantity(lc[colname].value, "electron/s")
+            else:
+                lc[colname].unit = "electron/s"
+
+    for colname in ["flux_bkg"]:
+        if colname in lc.colnames:
+            lc[colname].unit = u.percent
+
+    for colname in ["centroid_col", "centroid_row", "x_centroid", "y_centroid", "x_com", "y_com"]:
+        if colname in lc.colnames:
+            lc[colname].unit = u.pix
+
+    for colname in ["barycorr"]:
+        if colname in lc.colnames:
+            lc[colname].unit = u.day
+
+    # In Eleanor fits file, raw_flux's error is in flux_err, which breaks Lightkurve convention.
+    # To account for this, the corr_flux error is calculated from corr_flux_err = corr_flux*raw_flux_err/raw_flux. For completeness,
+    # the original raw_flux's error is added as a "raw_flux_err" column
+    lc["raw_flux_err"] = lc["flux_err"]
+    if flux_column.lower() != 'raw_flux':
+        lc["flux_err"] = lc[flux_column.lower()]*lc["raw_flux_err"]/lc["raw_flux"]
+
+    lc.meta["AUTHOR"] = "GSFC-ELEANOR-LITE"
+
+    # Eleanor light curves are not normalized by default
+    lc.meta["NORMALIZED"] = False
+
+    tic = lc.meta.get("TIC_ID")
+    if tic is not None:
+        # compatibility with SPOC, QLP, etc.
+        lc.meta["TARGETID"] = tic
+        lc.meta["TICID"] = tic
+        lc.meta["OBJECT"] = f"TIC {tic}"
+        # for Lightkurve's plotting methods
+        lc.meta["LABEL"] = f"TIC {tic}"
+
+    return TessLightCurve(data=lc, **kwargs)
```

Comment:
With `**kwargs` removed, it needs to be taken out.

Reply:
Good catch, parameter removed